MMM-COVID19.js

```diff
@@ -288,7 +289,7 @@ Module.register("MMM-COVID19", {
       // convert API date/time UTC to local timezone
       let dateToLocalTimezone = new Date(this.countriesStats['statistic_taken_at'] + ' UTC')
 
-      statsDateCell.innerHTML = this.translate('statistic taken at ') + dateToLocalTimezone
+      statsDateCell.innerHTML = this.translate('statistic taken at ') + dateToLocalTimezone.toLocaleString()
```

Comment:
@radokristof an ongoing PR (https://github.com/bibaldo/MMM-COVID19/pull/36) is in process of approval with these kind of changes (time and date format). So please remove this from your PR

Reply:
I just saw that now. I will remove this.